eth_client/test/eth_client_test.exs

```diff
@@ -1,4 +1,48 @@
 defmodule EthClientTest do
   use ExUnit.Case
   doctest EthClient
+  alias EthClient.Account
+  alias EthClient.Context
+
+  @bin "../contracts/src/bin/Storage.bin"
+  @abi "../contracts/src/bin/Storage.abi"
+
+  describe "Ethereum war tooling functions" do
+    test "Deploy" do
+      bin = @bin
+      abi = @abi
+
+      contract = EthClient.deploy(bin, abi)
+      assert contract == EthClient.Context.contract()
+    end
+
+    test "Invoke" do
+      bin = @bin
+      abi = @abi
+
+      contract = EthClient.deploy(bin, abi)
+      {:ok, tx_ans} = EthClient.invoke("store(uint256)", [3], 0)
+      assert tx_ans != nil
```

Comment:
I can't see the relationship between the contract and the `invoke/3` function, so... I think you don't need to use a contract or maybe the design is wrong and less usable than you think. In that case, you have to pass the contract as a parameter, remember that in functional programming state is not persisted, so the function has to be executed without dependencies on the module.

I think we would add more test to fail here.